earth_enterprise/SConstruct

```diff
@@ -26,6 +26,8 @@ EnsureSConsVersion(0, 96, 92)
 
 # process commandline arguments
 installdir = ARGUMENTS.get('installdir', '/tmp/fusion_os_install')
+with open('rpms/build.properties', 'w') as f:
+    print >> f, str('prefix=') + installdir.strip()
```

Comment:
Minor: I'd probably name this something a little more descriptive than "prefix", which is pretty generic. Internally it's called "stagedInstall_path", so that would probably be better.

Reply:
My original thought is we might later have "prefix=", "version=", and "release=", so I was thinking in terms of short words....

Reply:
"staged=" may be clearer than prefix= though...

Reply:
This one is minor, so it's really a matter of opinion. If you want to keep it to a single word, prefix or staged would be fine.

Reply:
I am renaming it to staged= as I think it is clearer given how it is used later.

Reply:
I vote for using the same name that's used in the `scons` scripts, and Gradle scripts.  It just makes it worse to rename a variable just to pass it from one place to another.  It makes it more tedious to track where a value is used.

Reply:
I hope we don't add `version` and `release` to this file.  Gradle can read the already existing version files from where they are.